CHANGELOG.md

```diff
@@ -5,7 +5,90 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 
-## [1.6.0] - 2022-MM-DD
+## [1.7.0] - 2022-MM-DD
+
+### Added
+
+-
+
+
+-
+
+
+-
+
+
+-
+
+
+-
+
+### Changed
+
+-
+
+
+-
+
+
+-
+
+
+-
+
+
+-
+
+### Deprecated
+
+-
+
+
+-
+
+
+-
+
+
+-
+
+
+-
+
+### Removed
+
+-
+
+
+-
+
+
+-
+
+
+-
+
+
+-
+
+### Fixed
+
+-
+
+
+-
+
+
+-
+
+
+-
+
+
+-
+
+
+## [1.6.0] - 2022-03-29
 
 ### Added
```

Comment:
I guess that the double blank line shall be also removed before the release...

Reply:
Yes. Not doing it here to avoid conflicts to open PRs